widget/view.controller.js

```diff
@@ -9,23 +9,51 @@
+  threatIntelManagementConfiguration100Ctrl.$inject = ['$scope', 'threatIntelManagementConfigurationService', 'WizardHandler', '$controller', '$state', 'connectorService', 'currentPermissionsService', 'CommonUtils', 'API', '_', '$filter', '$q', 'dataIngestionService', 'PagedCollection', '$resource', 'FIXED_MODULE', 'Entity', 'playbookService', 'translationService', 'toaster', 'appModulesService', 'widgetBasePath', '$rootScope'];
 
-  function threatIntelManagementConfiguration100Ctrl($scope, threatIntelManagementConfigurationService, WizardHandler, $controller, $state, connectorService, currentPermissionsService, CommonUtils, $window, API, _, $filter, $q, dataIngestionService, PagedCollection, $resource, FIXED_MODULE, Entity, playbookService, translationService, toaster, appModulesService, widgetBasePath, $rootScope, marketplaceService) {
+  function threatIntelManagementConfiguration100Ctrl($scope, threatIntelManagementConfigurationService, WizardHandler, $controller, $state, connectorService, currentPermissionsService, CommonUtils, API, _, $filter, $q, dataIngestionService, PagedCollection, $resource, FIXED_MODULE, Entity, playbookService, translationService, toaster, appModulesService, widgetBasePath, $rootScope) {
     $controller('BaseConnectorCtrl', {
       $scope: $scope
     });
     $scope.backToStartPage = backToStartPage;
+    $scope.moveVersionControlNext = moveVersionControlNext;
+    $scope.backToConfigConnector = backToConfigConnector;
     $scope.moveToConfigureConnector = moveToConfigureConnector;
     $scope.backToinstallConnector = backToinstallConnector;
     $scope.moveToSelectConnector = moveToSelectConnector;
+    $scope.backToSelectConnector = backToSelectConnector;
+    $scope.moveToFeedRules = moveToFeedRules;
+    $scope.backFeedRules = backFeedRules;
+    $scope.configFieldChanged = configFieldChanged;
+    $scope.moveToFinish = moveToFinish;
     $scope.installConnector = installConnector;
     $scope.toggleSelectFeedsSettings = toggleSelectFeedsSettings;
     $scope.toggleConnectorConfigSettings = toggleConnectorConfigSettings;
+    $scope.paramsUpdating = false;
+    $scope.feedConfigured = false;
+    $scope.toggleConnectorConfig = [];
+    $scope.toggleParametersConfig = [];
+    $scope.toggleScheduleConfig = [];
+    $scope.toggleParametersSettings = toggleParametersSettings;
+    $scope.toggleScheduleSettings = toggleScheduleSettings;
+    $scope.toggleScheduleConfigSettings = toggleScheduleConfigSettings;
+    $scope.saveParams = saveParams;
     $scope.allConnectorsInstalled = false;
+    $scope.loadActiveTab = loadActiveTab;
     $scope.feedConnectors = [];
+    $scope.isConnectorHealthy = [];
     $scope.installedConnectors = [];
+    $scope.samplePlaybookEntity = {};
+    $scope.ingestMethodActions = {};
     $scope.searchQuery = '';
+    $scope.availableConnectors = [];
+    $scope.healthyConnectorsParams = [];
+    $scope.connectorHealthStatus = [];
```

Comment:
one connector or multiple? these names should be more meaningful so that you can get its usage